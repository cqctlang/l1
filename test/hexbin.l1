
@include <strings.cqct>


s = "123456789abcdefghijklmnopqrstuvwxyz";
sh = bintohex(s);
shs = hextobin(sh);
printf("%s\n%s\n%s\n",s,sh,shs);

b = "\001\002\003\000\004\005\006\007\777\776\775\774\222";
bh = bintohex(b);
bhb = hextobin(bh);
printf("%a\n",bhb == b);


