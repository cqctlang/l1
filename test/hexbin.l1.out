123456789abcdefghijklmnopqrstuvwxyz
3132333435363738396162636465666768696a6b6c6d6e6f707172737475767778797a
123456789abcdefghijklmnopqrstuvwxyz
1
